opts/opts.go

```diff
@@ -130,6 +172,15 @@ func (opts *ListOpts) Len() int {
 type ValidatorFctType func(val string) (string, error)
 type ValidatorFctListType func(val string) ([]string, error)
 
+func ValidateLogOpts(val string) (string, error) {
+	allowedKeys := map[string]string{}
+	vals := strings.Split(val, "=")
+	if allowedKeys[vals[0]] != "" {
```

Comment:
`allowedKeys` is intended to contain the keys. Right now, there are no keys defined. Once someone adds a new key,  they will have to add that key into this map, otherwise, the validator will also reject that key.